README.md

```diff
@@ -23,21 +23,19 @@ as **T-F-D**:
  | 4           | signature          |
  | 5           | encrypted          |
  | 6           | generic            |
+ | 7           | identity           |
 
 ### 0. Feed ID formats
 
 A feed ID TFD represents the public portion of a cryptographic keypair used to
-identify a peer, and in turn identify feeds. Note however that there are some
-identities that do not have a feed nor create messages, such as Fusion
-Identities.
+identify a feed, and verify message signatures.
 
 | Type code | Format code | Data length | Format name     | Specification    |
 |:---------:|:-----------:|-------------|-----------------|------------------|
 | 0         | 0           | 32 bytes    | classic         | [classic]        |
 | 0         | 1           | 32 bytes    | gabby-grove     | [gabby grove]    |
 | 0         | 2           | 32 bytes    | bamboo          | [bamboo]         |
 | 0         | 3           | 32 bytes    | bendy-butt      | [bendy butt]     |
-| 0         | 4           | 32 bytes    | fusion-identity | [fusionidentity] |
```

Comment:
removed this because it's not fully formed yet, and we also suspect it should live in the new "identity" type.
I'm also interested in whether bamboo should be removed for the moment - it doesn't say here where recipe of bamboo feed...

Reply:
I'm all for adding things when we have a concrete implementation. I would say that fusion identity is not there yet (so fine adding when we get there), but gabby  & bamboo are, so would like to keep them in.